Libraries/LinkingIOS/LinkingIOS.js

```diff
@@ -76,6 +76,22 @@ var DEVICE_NOTIF_EVENT = 'openURL';
  * }
  * ```
  *
+ * The payload sent to JS in `_handleOpenURL` will be in the following format:
```

Comment:
`_handleOpenURL` looks like a private method. So I think we should not specify the name.